src/types/tmdb.ts

```diff
@@ -319,9 +323,22 @@ export interface ShowData {
     ];
 }
 
-export interface RecommendationData {
+/**
+ * Returned by MovieDB movie recommendation query
+ */
+export interface MovieRecommendationData {
     page?: number;
     results?: MovieData[];
     total_pages?: number;
     total_results?: number;
 }
+
+/**
+ * Returned by MovieDB tv recommendation query
+ */
+export interface TvRecommendationData {
```

Comment:
Same thing here for `TvShowData`.

Reply:
TvShowData is contained in this response. We could use `getTvByName` though

Reply:
Updated